types/util/validation.d.ts

```diff
@@ -0,0 +1,44 @@
+export declare function eq(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function gt(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function notEmpty(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function notEmpty(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function isIn(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function isSimple(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function isEmail(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function isUrl(
+    value?: unknown,
+    ctx?: any
+): Boolean;
+
+export declare function isRegex(
+    value?: unknown,
+    ctx?: any
+): Boolean;
```

Comment:
I believe these types are not correct. All these functions:
- Receive either 1 (message) or 2 (valueC, message) arguments
- Return a function that receive a (value, ctx) and return a boolean. The `value` should be typed